sphinx/cmd/build.py

```diff
@@ -142,7 +142,8 @@ def get_parser() -> argparse.ArgumentParser:
     group.add_argument('-d', metavar='PATH', dest='doctreedir',
                        help=__('path for the cached environment and doctree '
                                'files (default: OUTPUTDIR/.doctrees)'))
-    group.add_argument('-j', metavar='N', default=1, type=jobs_argument, dest='jobs',
+    group.add_argument('-j', '--parallel', metavar='N', default=1,
```

Comment:
Why not `--jobs` given the `dest`, etc.?

Reply:
(N.B., see also the extensive discussion above on that point)